src/components/common/AgreementType.vue

```diff
@@ -1,11 +1,11 @@
 <template>
-  <v-card flat id="agreement-summary review-header" class="rounded-0">
-    <div class="mt-4" v-if="isSummary">
+  <v-card flat id="agreement-summary" class="border-0">
```

Comment:
Typo. Will fix or remove.

(There are several instances. I need to test to see if this is even needed.)